src/components/TodoComponents/TodoForm.js

```diff
@@ -1,5 +1,14 @@
 import React from "react";
 
-const Form = () => <input type="text" placeholder="Add todo. . ." />;
-
+const Form = props => {
+  return (
+    <form>
+      <input value={props.newTodo} onChange={props.changeHandler} type="text" placeholder="Add todo. . ." />
+      <button type="submit" onClick={props.addNewTodo}>
+        Add Todo
+      </button>
+      <button>Clear Completed</button>
```

Comment:
clear button is refreshing the page. default behavior should be prevented on thursday